src/themes/shared/sizes.ts

```diff
@@ -1,4 +1,5 @@
 export const sizes = {
+  xs: '1rem',
```

Comment:
I have checked with Enrique and we need to introduce this new token for the checkbox size. It's already added it in figma I just need to update in the figma-tokens.